examples/resources/biganimal_cluster/single_node/resource.tf

```diff
@@ -63,7 +63,7 @@ resource "biganimal_cluster" "single_node_cluster" {
   pg_version            = "14"
   private_networking    = false
   cloud_provider        = "azure"
-  read_only_connections = false
+  read_only_connections = true
```

Comment:
This change is for testing the API Errors. 
It must be removed before merging this PR.